idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -595,16 +693,17 @@ def _therm_cond(self):
         # Thermal conductivity of gas
         self.therm_cond = Var(domain=Reals,
                               initialize=1e-5,
-                              doc="Thermal conductivity of gas [kJ/m.K.s]",
-                              units=pyunits.kJ/pyunits.m/pyunits.K/pyunits.s)
+                              doc="Thermal conductivity of gas [J/m.K.s]",
+                              units=pyunits.J/pyunits.m/pyunits.K/pyunits.s)
 
         def therm_cond_comp(i):
-            return self._params.therm_cond_param[i, 1] \
-                    * (self.temperature**self._params.therm_cond_param[i, 2]) \
-                    / ((1 + (self._params.therm_cond_param[i, 3] /
+            return (self._params.therm_cond_param_1[i] *
+                    ((self.temperature/pyunits.K) **  # need /K to cancel units
```

Comment:
Division by pyunits.K will no longer be necessary when the units for therm_cond_param_1 are updated per prior comment.